app/helpers/content_helper.rb

```diff
@@ -19,4 +19,14 @@ def is_current?(page)
   def is_mobile_menu_section_current?(page)
     params[:slug] == page.slug || params[:section] == page.slug
   end
+
+  def insert_block(block_name)
+    block = ContentBlock.find_by_name(block_name)
+    html_to_use = "Error - block not found"
+    if block
+      html_to_use = ContentBlock.find_by_name(block_name).markdown
+    end
```

Comment:
What are we doing the find twice? could be something like:

` html_to_use = block ? block.markdown : "Error - block not found"`

Reply:
Thanks !